pkg/dialer/dialer_windows.go

```diff
@@ -19,21 +19,13 @@ package dialer
 import (
 	"net"
 	"os"
-	"syscall"
 	"time"
 
 	winio "github.com/Microsoft/go-winio"
 )
 
 func isNoent(err error) bool {
-	if err != nil {
-		if oerr, ok := err.(*os.PathError); ok {
-			if oerr.Err == syscall.ENOENT {
-				return true
-			}
-		}
-	}
-	return false
+	return os.IsNotExist(err)
```

Comment:
This change is related to windows platform or go version?

Reply:
Not sure I understand the question. This change is just a simplification. `os.IsNotExist` [unwraps](https://golang.org/src/os/error.go#L126) `*os.PathError` so there's no point in doing it manually.

This function is needed when the `ENOENT` is wrapped inside `*net.OpError` (i.e. on unix)

Reply:
go1.13 starts to support wrap https://golang.org/doc/go1.13#error_wrapping. containerd don't add limitation of golang right now. If someone uses go1.12 to build it, it might be issue. That is what I said that it is related to golang version.